packages/swapkit/tokens/fetchTokenLists.ts

```diff
@@ -26,20 +26,21 @@ for (const { provider } of providers) {
 
   try {
     const tokenList = await RequestClient.get<TokensResponse>(
-      `https://static.thorswap.net/token-list/${provider}.json`,
+      `https://dev-api.swapkit.dev/tokens?provider=${provider.toUpperCase()}`,
```

Comment:
This will have to change before merge. either api v2 will be supporting all provider or v1 will have to add short codes